soweego/commons/db_manager.py

```diff
@@ -42,10 +41,10 @@ def __init__(self):
         password = credentials[constants.PASSWORD]
         host = credentials[constants.HOST]
         try:
-            # Disable connection pooling, as per Wikimedia policy
+            # Should disable connection pooling, as per Wikimedia policy
             # https://wikitech.wikimedia.org/wiki/Help:Toolforge/Database#Connection_handling_policy
             self.__engine = create_engine(
-                '{0}://{1}:{2}@{3}/{4}'.format(db_engine, user, password, host, db_name))
+                '{0}://{1}:{2}@{3}/{4}'.format(db_engine, user, password, host, db_name), pool_recycle=1800)
```

Comment:
Could you explain what's `pool_recycle` exactly doing here?

Reply:
`pool_recycle` sets a maximum connection age, after when it is destroyed and recreated. This helps to avoid to be disconnected by the database itself. We gain more control over our connection.
https://docs.sqlalchemy.org/en/latest/core/pooling.html#setting-pool-recycle